src/javascript/app/base/client.js

```diff
@@ -13,7 +13,7 @@ const getPropertyValue   = require('../../_common/utility').getPropertyValue;
 const Client = (() => {
     const processNewAccount = (options) => {
         if (ClientBase.setNewAccount(options)) {
-            window.location.href = options.redirect_url || defaultRedirectUrl(); // need to redirect not using pjax
+            setTimeout(() => {window.location.replace(options.redirect_url || defaultRedirectUrl());},250); // need to redirect not using pjax
```

Comment:
Why do we need a timeout here?

Reply:
Because without a timeout, in some cases on IOS and safari, a new page is opened before the location is replaced, so it opens the previous page instead of a successful message. That's the solution proposed on this thread: https://stackoverflow.com/questions/31223216/why-isnt-window-location-href-not-forwarding-to-page-using-safari